src/services/date-utilities.ts

```diff
@@ -360,4 +360,45 @@ export default class DateUtilities {
   static isDateFuture (date: FormattedDateTimeGmt): boolean {
     return (new Date(date) > new Date())
   }
+
+  /**
+   * Add a number of months to a date and return "YYYY-MM-DD".
+   * Date must be in the format of "YYYY-MM-DD" and months is a number
+   * @example (3, 2023-02-03) -> "2023-05-03"
+   * @example (18, 2023-02-03) -> "2024-08-03"
+   */
+  static addMonthsToDate (month: number, date: string): string {
+    if (!date) {
+      date = '2023-01-01'
+    }
```

Comment:
Is this ever called with date falsy? If not, this fallback could fail silently and I recommend against it.

PS For default values, you can also write
```
static addMonthsToDate (month: number, date = '2023-01-01'): string {
```

Reply:
Yes, I've double checked this. In the getter on line 238 (`extensionExpiryMin`), it's being called I believe before the incorporation date is properly set. Without the above code, I'm getting this:
![date falsy](https://github.com/bcgov/business-filings-ui/assets/122301442/06011f47-aaad-44c0-ada4-21034dcaeb1d)

I've tried console logging in the getter and same thing, it was being called twice. The first one is what's causing the issue.

Reply:
I think it would be acceptable to return null if the formal parameters are invalid.

I think the caller should provide a fallback if the value is null -- not this function, which shouldn't know about default dates.

Leo, can you update this, please?